jobs/uaa/templates/config/log4j2.properties.erb

```diff
@@ -15,7 +15,7 @@ case logging_format_timestamp_value
 end
 %>
 property.log_directory = /var/vcap/sys/log/uaa
-property.log_pattern=[<%= timestamp_format %>] uaa%X{context} - %pid [%t] .... %5p --- %c{1}: %replace{%m}{(?<=password=|client_secret=)([^&]*)}{<redacted>}%n
+property.log_pattern=[<%= timestamp_format %>] uaa%X{context} - %pid [%t] - [%X{traceId},%X{spanId}] .... %5p --- %c{1}: %replace{%m}{(?<=password=|client_secret=)([^&]*)}{<redacted>}%n
```

Comment:
This is technically a breaking change. Do we want to make it configurable in the spec? @strehle any thoughts?

Reply:
It could possibly break a user's log parser. Although it would be pretty easy to fix.

We don't really want to make it configurable, we think it's OK.

Reply:
Anything that changes the log format could break existing code that parses the logs. We asked around internally and no one was concerned about it as long as we mentioned the change in the release notes - it seems that our customers are used to updating their log parsing when they install an update. 

So I'm inclined not to make this configurable unless someone has a solid need for it.

Reply:
It appears this is resolved at this point?